### DISCUSSION TITLE
Is there any event information that fires when the done button is clicked in the data picker component?

### DISCUSSION THREAD

**Discussion by ryuhangyeong** [mantinedev/mantine#4706] [Q&A] [answered]
Is there an event that receives the changed date only when the area of ​​the photo is clicked after selecting the default value from the datetime picker?

![스크린샷 2023-08-30 오전 8 29 03](https://github.com/mantinedev/mantine/assets/27342882/fb7b1f96-8ac7-4acd-b834-8f1505f90d1c)

**Comment by rtivital** [comment]
You can add events to the submit button with `submitButtonProps` prop

**Reply by ryuhangyeong** [reply]
@rtivital Thanks! 

Is there any way to get the changed date value in that event? In my requirement, it is difficult to change the value everytime in onchange.

**Reply by rtivital** [reply, marked as answer]
You can store the value in state and reference it in one of the handlers
